### ISSUE TITLE
flux dev finetune, training loss drops from 0.5 and stays at around 0.34 for thousands of steps, no further changes

### ISSUE THREAD

**Issue by zzl1122** [kohya-ss/sd-scripts#1733]
I used the flux_train.py from SD3 branch to finetune Flux.1-dev model based on my own datasets which include 10k images. bs = 10. However, the training loss stays at 0.34 and doesn't drop anymore. I find the similar results no matter the lr=1e-6 or 5e-5. Can anyone show some helpful instructions?

**Comment by kelisiya** [comment]
Is there any further improvement in performance?

**Comment by zzl1122** [comment]
> Is there any further improvement in performance?

not yet
